upgrade_groupd.sh

```diff
@@ -0,0 +1,372 @@
+  echo "Data preparing for DEVEXP team was finished!"
+
+}
+
+#Monitoring
+function monitoring {
+  echo "Collectting pre_upgrade data for Monitoring team.... "
+  echo -e "================== MONITORING-TEAM  DATA `date` ============================================" >> $RESULT_FILE
+  echo  -e "#oc get clusteroperator monitoring -o yaml\n$(oc get clusteroperator monitoring -o yaml)" >> $RESULT_FILE
+  echo "=================Preparing upgrade data for MONITORING team....."
+  oc create -f https://raw.githubusercontent.com/juzhao/monitoring/master/config.yaml
+  echo "waiting for PVC creation....."
+  sleep 20
+  echo  -e "#oc -n openshift-monitoring get pvc | grep -v NAME | wc -l\n$(oc -n openshift-monitoring get pvc | grep -v NAME | wc -l)" >> upgradeData
+  echo -e "#for pod in \$(oc -n openshift-monitoring get pod  | grep prometheus-k8s | awk '{print \$1}'); do echo \$pod; oc -n openshift-monitoring get pod \$pod -oyaml | grep -i retention; done\n$(for pod in $(oc -n openshift-monitoring get pod  | grep prometheus-k8s | awk '{print $1}'); do echo $pod; oc -n openshift-monitoring get pod $pod -oyaml | grep -i retention; done)" >> $RESULT_FILE
+  echo -e "#oc -n openshift-monitoring logs \$(oc -n openshift-monitoring get pod | grep telemeter-client | awk '{print \$1}') -c telemeter-client | grep "https://infogw.api.stage.openshift.com"\n$(oc -n openshift-monitoring logs $(oc -n openshift-monitoring get pod | grep telemeter-client | awk '{print $1}') -c telemeter-client | grep https://infogw.api.stage.openshift.com)" >> $RESULT_FILE
+  echo "Data preparing for MONITORING team was finished!"
+}
+
+#Logging team
+
+function enableLogging {
+  echo "==================Install LOGGING........"
+  oc create -f https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/logging/clusterlogging/OCP-21311/namespace.yaml
+  oc create -f https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/logging/clusterlogging/OCP-21311/operator-group.yaml
+  oc create -f https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/logging/clusterlogging/OCP-21311/csc-clusterlogigng.yaml
+  oc create -f https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/logging/clusterlogging/OCP-21311/csc-elasticsearch.yaml
+  oc create -f https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/logging/clusterlogging/OCP-21311/sub-cluster-logging.yaml
+  oc create -f https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/logging/clusterlogging/OCP-21311/sub-elasticsearch-operator.yaml
+  sleep 30
+
+  echo "==================waiting for pods running....."
+  oc create -f https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/logging/clusterlogging/example.yaml  -n openshift-logging
+  sleep 30
+  echo "==================waiting for pods running....."
+  if [ !$(oc get po -n openshift-logging | tail -n +2 |grep  -cvE "(Running|Completed)") ] ; then
```

Comment:
@SSshahan   if there are running elasticsearch, kibana and fluentd pod, it pass the test. (the curator pods may be in error status before the Elasticsearch is not ready. )
oc get pods --selector component=elasticsearch --n openshift-logging
oc get pods --selector component=kibana --n openshift-logging
oc get pods --selector component=fluentd --n openshift-logging

Reply:
@SSshahan, thanks for starting this effort.  Since this is a public repo, I don't think this is the correct place for this.  Perhaps either `verification-test` or `openshift-tests`,  WDYT @akostadinov?

Reply:
Not sure how this script will be used. In case it has hardcoded some internal stuff thus making it useless to the community, then yes - internal private repo be used.
In case it or parts of it could be useful for somebody, then it would be fine to keep public in my opinion.
I don't have a strong opinion here.

Reply:
@akostadinov, I believe this script is for preparing various environment for upgrade testing for ` `group_d` within `OpenshitQE`, that's why I think it should've be made public since it's a helper for internal testing of the product (I doubt our external orgs care about this).

Reply:
If it doesn't hardcode something only relevant to our internal network and setup, then it's fine either way. I fully agree though that writing a step to execute these commands would be much nicer.

Reply:
It's fine to keep it public if no specific secure, and we do need write function and split test files out if still use `shell`